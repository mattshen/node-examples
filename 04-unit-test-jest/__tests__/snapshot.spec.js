
function hello() {
    return {
        to: "matt",
        date: "12/12/2019",
        message: "hello, how are you?"
    }
}

test("should return proper greeting", () => {
    expect(hello()).toMatchSnapshot();
})
