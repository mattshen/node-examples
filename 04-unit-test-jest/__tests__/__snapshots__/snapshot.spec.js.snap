// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return proper greeting 1`] = `
Object {
  "date": "12/12/2019",
  "message": "hello, how are you?",
  "to": "matt",
}
`;
